docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/{{ cookiecutter.module_name }}/templates/{{ cookiecutter.module_name }}/partials/core_content.html

```diff
@@ -1,4 +1,4 @@
-{% load static %}
+{% raw %}{% load static %}
```

Comment:
why do we need these raw tags?

Reply:
Both Django and Cookiecutter use Jinja2 so the Cookiecutter will try to interpret Django template tags (e.g. `{% load static %}`) unless you explicitly tell it to just process them as plaintext.